lib/regexp_parser/scanner/scanner.rl

```diff
@@ -797,22 +798,29 @@ class Regexp::Scanner
   end
 
   # Emits an array with the details of the scanned pattern
-  def emit(type, token, text, ts, te)
+  def emit(type, token, text)
```

Comment:
Char indices are now tracked in this method, hence why ts, te no longer need to be passed